src/main/java/uk/gov/hmcts/et/common/model/ccd/CaseData.java

```diff
@@ -1123,7 +1125,7 @@ public class CaseData extends Et1CaseData {
     @JsonProperty("tseAdmReplyAdditionalInformation")
     private String tseAdmReplyAdditionalInformation;
     @JsonProperty("tseAdmReplyAddDocument")
-    private UploadedDocumentType tseAdmReplyAddDocument;
+    private List<GenericTypeItem<DocumentType>> tseAdmReplyAddDocument;
```

Comment:
Is this a breaking change?

Reply:
Yes, the ticket is to change these singular uploads into multiple